### PR TITLE
Return a boolean depending on whether or not truncation occurred

### PR DESCRIPTION
**Pull request by afreix**
Addresses https://github.com/dollarshaveclub/shave/issues/87

### CODE REVIEW

File: src/shave.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 export default function shave(target, maxHeight, opts = {}) {
   if (!maxHeight) throw Error('maxHeight is required')
+  var didTruncate = false
```
Comment (yowainwright) [inline src/shave.js]: - use `let`
- rename the variable`didShave`

**Comment by yowainwright** [comment]
@afreix are you gonna work on this feature? 😀 ~Thanks

**Comment by afreix** [comment]
Yes! Sorry I got caught up with work and life, but I will get to it tomorrow!

**Comment by yowainwright** [comment]
@afreix no worries at all. Take your time. I will appreciate seeing your vision. 👍

**Comment by yowainwright** [comment]
@afreix do you want to work on this or shall I close this PR? ~Thanks no matter what! 😊

**Pull request closed without merging**